src/datumaro/components/annotation.py

```diff
@@ -215,7 +216,32 @@ class Label(Annotation):
 @attrs(slots=True, eq=False, order=False)
 class HashKey(Annotation):
     _type = AnnotationType.hash_key
-    hash_key: np.ndarray = field(factory=lambda: np.zeros((1, 64), dtype=np.uint8))
+    hash_key: np.ndarray = field(validator=attr.validators.instance_of(np.ndarray))
+
+    @hash_key.validator
```

Comment:
Is there any reason to use validator for `HashKey`?

Reply:
I think that we don't need default factory. What we need to a validator which checks its input type, dim, dtype, and shape.

Reply:
Every logic requires `HashKey.hash_key` to be 1D numpy array with `dtype=np.uint8` and `shape[0]=64`, so that I think that it is better to check the input data when creating `HashKey`.